src/Plugins/SimplnxCore/src/SimplnxCore/Filters/ReadVtkStructuredPointsFilter.cpp

```diff
@@ -117,7 +130,7 @@ IFilter::PreflightResult ReadVtkStructuredPointsFilter::preflightImpl(const Data
   inputValues.PointAttributeMatrixName = filterArgs.value<DataObjectNameParameter::ValueType>(k_VertexAttributeMatrixName_Key);
   inputValues.CellAttributeMatrixName = filterArgs.value<DataObjectNameParameter::ValueType>(k_CellAttributeMatrixName_Key);
 
-  resultOutputActions = ReadVtkStructuredPoints::PreflightFile(inputValues);
+  resultOutputActions = ::PreflightFile(inputValues);
```

Comment:
Shouldn't this pass in the message handler and other arguments from the preflight function?

Reply:
Seeing as this was the case before this code was moved, this doesn't have to be addressed in this PR.